src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -370,6 +370,39 @@ module Types = struct
               Stringable.Ledger_hash.to_base58_check
               @@ Staged_ledger_hash.ledger_hash staged_ledger_hash ) ] )
 
+  let natural_number_desc field_name num_bits =
+    Core.sprintf
+      !"String encoded natural number of the %s (%i bits)"
+      field_name num_bits
+
+  let consensus_state =
+    let open Consensus.Data.Consensus_state in
+    obj "ConsensusState" ~fields:(fun _ ->
+        [ field "blockchainLength" ~typ:(non_null string)
+            ~doc:
+              (natural_number_desc "blockchain length"
+                 Coda_numbers.Length.length_in_bits)
```

Comment:
you can use uint32 for this too as length is a uint32 as well (we only have it in a separate module for type-safety reasons, not to override the bit length)